listenbrainz/webserver/views/user.py

```diff
@@ -253,3 +253,18 @@ def delete_user(musicbrainz_id):
         error_msg='Could not put user %s into queue for deletion, please try again later' % musicbrainz_id,
     )
     db_user.delete(user.id)
+
+def delete_listens_history(musicbrainz_id):
+    """ Delete a user's listens from ListenBrainz completely.
+    This, drops the user's influx measurement and resets their listen count. 
+
+    Args:
+        musicbrainz_id (str): the MusicBrainz ID of the user
+
+    Raises:
+        NotFound if user isn't present in the database
+    """
+
+    user = _get_user(musicbrainz_id)
```

Comment:
You need to catch and handle exceptions from here.

Reply:
@mayhem Thanks for the review. Most of the exceptions are handled in the `delete` function in _listenstore.py_ [here](https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/listenstore/influx_listenstore.py#L916). Can you give me an idea which exceptions can I handle in the function I created.

Reply:
Sorry, never mind this point.